pkgs/development/compilers/gcc/5/default.nix

```diff
@@ -547,4 +547,10 @@ stdenv.mkDerivation ({
 // optionalAttrs (!stripped || cross != null) { dontStrip = true; NIX_STRIP_DEBUG = 0; }
 
 // optionalAttrs (enableMultilib) { dontMoveLib64 = true; }
+
+# This is only here because it was here in previous commits and we
+# want to avoid a meaningless mass rebuild.  These variables get
+# overridden via crossAttrs in an actual cross-build.
+// { crossMingw = false; crossStageStatic = true; }
+
```

Comment:
I'm not in the position to merge but nixpkgs is a community project and I consider myself part of the community just like any other contributor :wink: 
This is my subjective view:
Avoiding a single mass rebuild does not weight out the cons of "dead code" (whereas during testing it is very valid), and one mass rebuild is not a blocker for pull requests to get merged.
Since we only need this code to make the test work, it should be moved into the test file.